### PR TITLE
New DHWT, Standard Calorifier with Return Pump

### PR DESCRIPTION
**Pull request by Ciaran-Oflannagain**
@cstirdivant - do you think the destratification pump is needed for this? Thanks

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/DHWT.yaml [hunk 1]
```diff
@@ -16,6 +16,16 @@
 ### Canonical Types  ###
 ########################
 
+DWHT_HTZTC_HWTTC_CPC_HWSWC:
+  description: "Standard calorifier with return pump."
+  is_canonical: true
+  implements:
+  - DHWT
+  - HTZTC
```
Comment (cstirdivant) [inline ontology/yaml/resources/HVAC/entity_types/DHWT.yaml:24]: This is the incorrect use for `HTZTC`; that type is typically used in air-side systems for an electric (or gas) heater that controls air temperature.

Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/DHWT.yaml]: Hi Claire - Apologies, the HTZTC was initially to cover the immersion electric heater (which is a backup and not the primary source source of heat) in this DHWT. I was unsure what abstract type to use, or if this point is needed in the cloud. I have removed the point. Thanks

Comment (cstirdivant) [reply ontology/yaml/resources/HVAC/entity_types/DHWT.yaml]: Hi @Ciaran-Oflannagain thanks for the clarification - if you do want to model the immersion heater, please use `HTSWC`

Comment (Ciaran-Oflannagain) [reply ontology/yaml/resources/HVAC/entity_types/DHWT.yaml]: Okay great thank you, I have added HTSWC.

**Comment by Ciaran-Oflannagain** [comment]
@cstirdivant requested changes have been completed, thanks

**Review by cstirdivant — APPROVED** [review]

**Pull request merged**